app/main.py

```diff
@@ -1,7 +1,39 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def get_weather() -> str:
```

Comment:
The function signature of `get_weather` should reflect the correct return type. It currently returns a string, so the signature `-> str` is correct. Please ensure this matches the task requirements.